samples/nrf9160/location/overlay-nrf7002ek-wifi-scan-only.conf

```diff
@@ -0,0 +1,54 @@
+#Overlay to use nRF7002EK on top of nrf9160DK for Wi-Fi scanning
```

Comment:
License header is missing. And minor nitpicks:
```suggestion
# Overlay to use nRF7002 EK on top of nRF9160 DK for Wi-Fi scanning
```

Reply:
Changed